southworkscom/SOUTHWORKS-azure-pipelines-tasks
# [ApiCompat] Add Unit Tests for the task

- Create unit tests to test task correct functionality
  - Add test for when no issues are found
  - Add test for when issues are found, but task should not fail
  - Add test for when issues are found, and task should fail
  - Add test for when a baseline file is used
  - Add test for when the log generation is checked
